test/evergreen/code_change_report/git_diff_tool.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+
+#
+# The code in this file is used to generate diff files that exclude any newly added 0-length files
+# as loading diff files that contain newly added 0-length files does not work in pygit2.
+#
+# Note: 0-length means empty files with no content. Files containing only blank lines are not 0-length, and
+# diffs including such files work in pygit2.
+#
+
+import argparse
+import glob
+import os
+import subprocess
+from typing import List
+from pygit2 import discover_repository, Repository
+from push_working_directory import PushWorkingDirectory
+
+
+def run_command(directory: str, command: str) -> str:
+    working_directory = PushWorkingDirectory(directory)
+    completed_process = subprocess.run(command, capture_output=True, check=True, shell=True)
+    output = completed_process.stdout
+    working_directory.pop()
+    return output.decode().strip()
+
+
+def find_zero_length_files(directory: str) -> List[str]:
+    working_directory = PushWorkingDirectory(directory)
+    files = glob.glob('./**/*'.format(directory), recursive=True)
```

Comment:
```suggestion
    files = glob.glob('./**/*', recursive=True)
```

Reply:
Hi Andrew, that's a leftover after refactoring. I'll fix it.